### ISSUE TITLE
Inconsistency  between .seen calls.

### ISSUE THREAD

**Issue by hadson172** [pingwindyktator/pybot#4]
When we call `seen i` there is output as shown bellow:
```
[22:31] <Interpod> .seen i
[22:31] <@botpingwina> Interpod was last seen 0 seconds ago saying ".seen i"
```

but when you write:
```
[22:34] <Interpod> .seen Interpod
[22:34] <@botpingwina> o rly?
```

As you can see there is inconsistency between bot responses.. This could be source of more seriously issue.

**Comment by pingwindyktator** [comment]
That's indeed a problem, thanks!

**Comment by pingwindyktator** [comment]
Fixed in 9c85c444b105b47cdbbcbff5a7740a5e24fa0538
